### PR TITLE
Set current dir at check_wasm_toolchain_installed at wasm-builder

### PR DESCRIPTION
**Pull request by MOZGIII**
This is an addition to #9926, https://github.com/paritytech/substrate/pull/9926#issuecomment-970340431.

This takes care of the case where the project root contains `.cargo` dir, and the code is executing in Github Actions. Long story short, for some reason with this combination, the project's `.cargo/config.toml` is picked up.
This change makes it work.

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

File: utils/wasm-builder/src/prerequisites.rs [hunk 1]
```diff
@@ -120,6 +120,7 @@ fn check_wasm_toolchain_installed(
 	let manifest_path = temp.path().join("Cargo.toml").display().to_string();
 
 	let mut build_cmd = cargo_command.command();
+	build_cmd.current_dir(&temp);
```
Comment (bkchr) [inline utils/wasm-builder/src/prerequisites.rs:123]: Could you please add a comment here for what this is.

**Comment by bkchr** [comment]
And please merge master to make cargo fmt happy.

**Comment by MOZGIII** [comment]
Code review comments addressed, ready to proceed.

**Comment by bkchr** [comment]
bot merge

**Pull request merged**